src/Components/ImagesTable/ImagesTable.tsx

```diff
@@ -81,6 +83,14 @@ const ImagesTable = () => {
   const blueprintsOffset = useAppSelector(selectOffset);
   const blueprintsLimit = useAppSelector(selectLimit);
 
+  let useGetBlueprintsQuery;
```

Comment:
Can we try doing a dynamic import at the top level instead of this? Something like this:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import#importing_different_modules_based_on_environment